terraform/stacks/umccr_data_portal/workflow/main.tf

```diff
@@ -189,6 +189,83 @@ locals {
   }
 }
 
+  transcriptome_wfl_id = {
+    dev  = "wfl.286d4a2e82f048609d5b288a9d2868f6"
+    prod = "wfl.286d4a2e82f048609d5b288a9d2868f6"
+  }
+
+  transcriptome_wfl_version = {
+    dev  = "3.7.5"
+    prod = "3.7.5--99abc32"
+  }
+
+  transcriptome_input = {
+    dev = <<-EOT
+    {
+      "sample-name": null,
+      "fastq_list_rows": null,
+      "output_file_prefix": null,
+      "output_directory": null,
+      "annotation_file": {
+            "class": "File",
+            "location": "gds://umccr-refdata-dev/dragen/hsapiens/hg38/rnaseq/ref-transcripts.non-zero-length.gtf"
+        },
+      "cytobands": {
+            "class": "File",
+            "location": "gds://umccr-refdata-dev/dragen/hsapiens/hg38/rnaseq/fusion/arriba-cytobands.tsv"
+        },
+      "blacklist": {
+            "class": "File",
+            "location": "gds://umccr-refdata-dev/dragen/hsapiens/hg38/rnaseq/fusion/arriba-blacklist.tsv.gz"
+        },
+      "protein_domains": {
+            "class": "File",
+            "location": "gds://umccr-refdata-dev/dragen/hsapiens/hg38/rnaseq/fusion/arriba-protein-domains.gff3"
+        },
+      "reference_fasta": {
+        "class": "File",
+        "location": "gds://umccr-refdata-dev/dragen/genomes/hg38/hg38.fa"
+      },
+      "reference_tar": {
+        "class": "File",
+        "location": "gds://umccr-refdata-dev/dragen/genomes/hg38/3.7.5/hg38_alt_ht_3_7_5.tar.gz"
+      }
+    }
+    EOT
+    prod = <<-EOT
+    {
+      "sample-name": null,
```

Comment:
Ditto `sample_name`

Reply:
I don't think sample name is an input into this workflow

```
$ cwltool --make-template workflows/dragen-transcriptome-pipeline/3.7.5/dragen-transcriptome-pipeline__3.7.5.cwl 2>/dev/null | yq eval --tojson
{
  "reference_tar": {
    "class": "File",
    "path": "a/file/path"
  },
  "reference_fasta": {
    "class": "File",
    "path": "a/file/path"
  },
  "protein_domains": {
    "class": "File",
    "path": "a/file/path"
  },
  "output_file_prefix": "a_string",
  "output_directory": "a_string",
  "fastq_list_rows": [
    {
      "rgsm": "a_string",
      "rglb": "a_string",
      "rgid": "a_string",
      "read_2": {
        "class": "File",
        "path": "a/file/path"
      },
      "read_1": {
        "class": "File",
        "path": "a/file/path"
      },
      "lane": 0
    }
  ],
  "enable_rna_quantification": false,
  "enable_rna_gene_fusion": false,
  "enable_map_align_output": false,
  "enable_duplicate_marking": false,
  "cytobands": {
    "class": "File",
    "path": "a/file/path"
  },
  "contigs": "a_string",
  "blacklist": {
    "class": "File",
    "path": "a/file/path"
  },
  "annotation_file": {
    "class": "File",
    "path": "a/file/path"
  }
}
```

Reply:
But just on this, we do now have a standard now for inputs and outputs that just allows lowercase, underscores and numbers for both workflows and tools. We've implemented this in the last iteration of cwl-ica and so are just teething out / updating any tools / workflows that violate this rule.

Reply:
Yes - we have switched to using `output_file_prefix`. I have removed `sample-name`.

Reply:
Thanks guys!